App.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    /*<div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>*/
+    <div>
+      <PageTitle />
+      <PageTasks /> 
+    </div>
+  );
+}
+
+export default App;
+
+//class page title
+class PageTitle extends React.Component{
+  render(){
+    return (
+      <div className="PageTitle-div">
```

Comment:
use kebab casing for css class name (e.g. `something-something-something`), and please add html tag type into the class name. Also since title is just one thing, not really a class of things. does it make more sense to use `id` instead of `className`